ajax/libs/peppermint/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "peppermint",
+  "version": "1.4.0",
+  "license": "MIT",
+  "filename": "peppermint.min.js",
+  "description": "Responsive touch-enabled slider. Tiny, supports mouse and all sorts of touch, library agnostic.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/wilddeer/Peppermint.git"
+  },
+  "author": {
+    "name": "Oleg Korsunsky",
+    "url": "https://wd.dizaina.net/"
+  },
+  "homepage": "https://wd.dizaina.net/en/scripts/peppermint/",
+  "keywords": [
+    "touch",
+    "touch-events",
+    "pointer-events",
+    "mouse-events",
+    "slider",
+    "carousel",
+    "gallery",
+    "vanilla-js",
+    "framework-agnostic",
+    "library-agnostic"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/wilddeer/Peppermint.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/!(*.+(pure.js|pure.min.js))"
```

Comment:
the result of this fileMap as the followings:
```
$ node
> glob = require('glob')
> glob.sync('**/!(*.+(pure.js|pure.min.js))')
[ 'peppermint.js',
  'peppermint.min.js',
  'peppermint.required.css',
  'peppermint.suggested.css',
  'pure' ]
```

I don't think we need the folder `pure`
How about this?
```
> glob.sync('**/!(*pure*)')
[ 'peppermint.js',
  'peppermint.min.js',
  'peppermint.required.css',
  'peppermint.suggested.css' ]
```